pandas/io/parsers.py

```diff
@@ -282,7 +282,7 @@
     "bad line" will be output.
 delim_whitespace : bool, default False
     Specifies whether or not whitespace (e.g. ``' '`` or ``'\t'``) will be
-    used as the sep. Equivalent to setting ``sep='\s+'``. If this option
+    used as the sep. Equivalent to setting ``sep='\\s+'``. If this option
```

Comment:
I think using raw strings is in general more readable: `sep=r'\s+'`. Just a comment, you can leave it like this.

Reply:
It was my first choice but it still gave me an error. Might be a bug in flake8.

Reply:
ok, let's not waste time on this at this point then